platforms/bukkit/common/src/main/java/com/dfsek/terra/bukkit/handles/BukkitWorldHandle.java

```diff
@@ -29,16 +29,28 @@
 import com.dfsek.terra.bukkit.world.block.data.BukkitBlockState;
 import com.dfsek.terra.bukkit.world.entity.BukkitEntityType;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 
 public class BukkitWorldHandle implements WorldHandle {
     private final BlockState air;
 
+    private static final Logger logger = LoggerFactory.getLogger(BukkitWorldHandle.class);
+
     public BukkitWorldHandle() {
         this.air = BukkitBlockState.newInstance(Material.AIR.createBlockData());
     }
 
     @Override
     public synchronized @NotNull BlockState createBlockState(@NotNull String data) {
+        if(data.equals("minecraft:grass")) { //TODO: remove in 7.0
```

Comment:
can this not be done somewhere globally in terra, rather than in each individual platform?

Reply:
it can be but that introduces minecraft specific code into core which is not really supposed to happen